rpcq/_client.py

```diff
@@ -56,18 +57,21 @@ def __init__(self, endpoint: str, timeout: float = None):
         # Cache of replies so that different tasks can share results with each other
         self._replies: Dict[str, Union[RPCReply, RPCError]] = {}
 
-    async def call_async(self, method_name: str, *args, **kwargs):
+    async def call_async(self, method_name: str, timeout: float = None, *args, **kwargs):
```

Comment:
I don't think this works: the first element of the `*args` tuple gets bound to `timeout`

Reply:
yuck, good point... is there python magic to work around that? or must I drop *args completely and just use *kwargs?

Reply:
(git marked as outdated, but this is still an issue)

Reply:
Not that I'm aware of. But with the `rpcq_timeout` solution we can just pop that it out of `kwargs` 
```suggestion
    timeout = kwargs.pop("rpcq_timeout", None) or self.timeout
```

Reply:
In Python 3 you can just do `call_async(self, method_name: str, *args, rpcq_timeout: float = None, **kwargs)` instead of the .pop

Reply:
Slick.

Reply:
I presume we are okay with pop since _call_async() doesn't use timeout